CentralityiOS/CentralityiOS/ViewControllers/ToDoFeedViewController.m

```diff
@@ -16,6 +16,8 @@ @interface ToDoFeedViewController () <UITableViewDelegate, UITableViewDataSource
 @property (weak, nonatomic) IBOutlet UIButton *logoutButton;
 @end
 
+NSInteger kToDoFeedLimit = 20;
```

Comment:
this should have `static const` in front of `NSInteger` (`const` to match the `k` prefix you have, and `static` to make sure it doesn't collide with variables in other modules)